packages/README.md

```diff
@@ -44,6 +44,7 @@ When creating a new package, you need to provide at least the following:
     - Package description
     - Installation details
     - Usage example
+    - API Docs, if applicable. See the "Maintaining API docs" section.
```

Comment:
```suggestion
    - API documentation, if applicable (see the "Maintaining API docs" section)
```

Reply:
Ideally, "Maintaining API docs" would link to `#maintaining-api-docs`.